### PR TITLE
[ignore] Annotations - try to merge master again

### PR DESCRIPTION
**Pull request by wild-endeavor**
basically ran
```
git co annotations
git pull
git co -b merge-again
git fetch origin
git merge origin/master
```
and resolved conflicts

### CODE REVIEW

File: flytekit/models/literals.py [hunk 1]
```diff
@@ -698,11 +586,131 @@ def from_flyte_idl(cls, pb2_object):
         :param flyteidl.core.literals_pb2.LiteralMap pb2_object:
         :rtype: LiteralMap
         """
-        return cls({k: Literal.from_flyte_idl(v) for k, v in pb2_object.literals.items()})
+        return cls({k: Literal.from_flyte_idl(v) for k, v in _six.iteritems(pb2_object.literals)})
+
+
+class Scalar(_common.FlyteIdlEntity):
```
Comment (kumare3) [inline flytekit/models/literals.py:592]: why is this here?

Comment (kumare3) [reply flytekit/models/literals.py:592]: nm it just moved

**Review by kumare3 — APPROVED** [review]

File: flytekit/models/literals.py (hunk 1 above)
Comment (wild-endeavor) [reply flytekit/models/literals.py:592]: the new file will be identical to master.  not sure what you mean, how did these files ever get here?  Or why is scalar defined here?
